Code/Editor/MainWindow.cpp

```diff
@@ -508,16 +508,6 @@ void MainWindow::Initialize()
     ActionOverrideRequestBus::Event(
         GetEntityContextId(), &ActionOverrideRequests::SetupActionOverrideHandler, this);
 
-    if (auto imGuiManager = AZ::Interface<ImGui::IImGuiManager>::Get())
-    {
-        auto handleImGuiStateChangeFn = [](bool enabled)
-        {
-            EditorWindowUIRequestBus::Broadcast(&EditorWindowUIRequests::SetEditorUiEnabled, enabled);
-        };
-        m_handleImGuiStateChangeHandler = ImGui::IImGuiManager::ImGuiSetEnabledEvent::Handler(handleImGuiStateChangeFn);
```

Comment:
Can also remove the `m_handleImGuiStateChangeHandler` member in `MainWindow` now that its unused, and I think the `ImGuiBus.h` include above

Reply:
Okay, I'll take care of that in my next PR.